runtimes/aleph-debian-11-python/init1.py

```diff
@@ -340,7 +357,7 @@ async def run_executable_http(scope: dict) -> Tuple[Dict, Dict, str, Optional[by
                 headers, body = await make_request(session, scope)
             except aiohttp.ClientConnectorError:
                 if tries > 20:
-                    raise
```

Comment:
Should the error still appear in the logs ?

Reply:
I don't think so, because cannot be an error, can be that the VM needs some time to be ready. When we implement the VM logs, we will see there if the VM is loading well or not.

Reply:
OK